timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/database/DataStoreOperations.java

```diff
@@ -218,6 +220,31 @@ private static Edge getExpectedEdge(GraphTraversalSource traversal, String timId
     }
   }
 
+  public void clearMappingErrors(String vreName) {
+    final GraphTraversal<Vertex, Vertex> collectionT = getRawCollectionsTraversal(vreName);
+    while (collectionT.hasNext()) {
+      final Vertex collection = collectionT.next();
+      collection.vertices(Direction.OUT, HAS_NEXT_ERROR).forEachRemaining(this::clearMappingErrors);
```

Comment:
This loops along all vertices whether they have errors or not. A faster approach would be to traverse along the error vertices as happens in RawCollection.java

```
    verticesWithErrors = collection.out(HAS_NEXT_ERROR)
      .emit()
      .repeat(__.out(HAS_NEXT_ERROR))
```

Also, I guess we'd need to remove the error property itself.